frontend/src/lib/components/PropertyFilters/components/PropertyValue.tsx

```diff
@@ -98,17 +102,33 @@ export function PropertyValue({
                 ...options,
             })
         } else {
-            api.get(endpoint || 'api/' + type + '/values/?key=' + key + (newInput ? '&value=' + newInput : '')).then(
-                (propValues: PropValue[]) => {
+            if (type.includes(TaxonomicFilterGroupType.Groups)) {
```

Comment:
I don't think we should have this branch at all here - we want this code to be as general as possible.

If you look at the original branch then this should be easily refactorable to allow configuration from within frontend/src/lib/components/TaxonomicFilter/taxonomicFilterLogic.ts